praw/models/auth.py

```diff
@@ -4,7 +4,8 @@
 
 from prawcore import Authorizer, ImplicitAuthorizer, UntrustedAuthenticator, session
 
-from ..exceptions import InvalidImplicitAuth, MissingRequiredAttributeException
+from praw.exceptions import InvalidImplicitAuth, MissingRequiredAttributeException
```

Comment:
Why the change to an absolute import?

Reply:
Simply because a rule exists for it: https://docs.astral.sh/ruff/rules/relative-imports/

Reply:
Ah, gotcha. I'm not sure I'm a fan of the mixing of relative and absolute imports.

Reply:
They should _all_ be absolute imports now. If not, the tool might not be doing its job.

Reply:
Oh hmm, I see that they aren't. That's strange.

Reply:
Ah I have to turn on this option: https://docs.astral.sh/ruff/settings/#lint_flake8-tidy-imports_ban-relative-imports